SyncRegistrationData.js

jeffbearer opened the pull request.

jeffbearer committed:
SyncRegistrationData.js

mihiralve commented on airtable-automations/SyncRegistrationData.js:
```diff
@@ -0,0 +1,205 @@
+// SyncEventAttendees.js
```
Can you update this to match the actual file name? Had this in here so that we know what file to look for from airtable

jeffbearer committed:
stopgap to fix registration sync

hit a corner case where an event registrant for a volunteer day is a guest
or family member of the actual member and the import broke.

I added a conditional to allow the script to run and warn on this
condition,  fixing this would be much more involved.
